src/IECoreHoudini/SOP_SceneCacheSource.cpp

```diff
@@ -311,8 +311,8 @@ void SOP_SceneCacheSource::loadObjects( const IECore::SceneInterface *scene, Ima
 	{
 		return;
 	}
-	
-	if ( scene->hasObject() && UT_String( scene->name() ).multiMatch( params.shapeFilter ) && tagged( scene, params.tagFilter ) )
+
+	if ( scene->hasObject() && UT_String( getFullScenePath( scene ) ).multiMatch( params.shapeFilter ) && tagged( scene, params.tagFilter ) )
```

Comment:
Also, I think we can maintain compatibility with existing scenes by multimatching both the full path and the short name, right?